### ISSUE TITLE
Reversed Gate list in Stack Explorer

### ISSUE THREAD

**Issue by metapone** [snebjorn/ttp-eldritch-horror#4]
If you open the Gate Stack's Stack Explorer, the gates will be displayed from the bottom instead of top.

**Comment by snebjorn** [comment]
That's how TTP displays them. I can't change that

**Comment by metapone** [comment]
That sounds like a bug more than a feature to be honest. Hope TTP's dev comes around and fix that.

**Comment by snebjorn** [comment]
A reversed card order for face down (flipped) decks have been requested for TTP. Now we play the waiting game.

Closing as there's nothing more I can do.

**Comment by snebjorn** [comment]
I'm curious though. Did you need to inspect the gate stack for a specific reason? I believe I scripted all interactions with it via custom scripts.

**Comment by metapone** [comment]
[Akachi's](https://eldritchhorror.fandom.com/wiki/Akachi_Onyele) special action allows her to look at the top 2 gates and put 1 to the bottom of the Gate Stack.

**Comment by snebjorn** [comment]
Right. But using the stack explorer you'd be able to see all of them :)
I just draw the 2 top gates and use shift to put one of the gates at the bottom of the stack after looking at them

**Comment by snebjorn** [comment]
Sort of. If you're already holding a card/token you can use right-click (while still holding a card/token) to pickup more items.
But in this case where it's just 2 gates. I'd just draw them individually and flip (<kbd>F</kbd>) immediately. Pretty much how I'd also doing it in the real world version :)
